Styles/badge.css

```diff
@@ -0,0 +1,78 @@
+.badge-items {
+  align-items: flex-end;
+  justify-content: center;
+  padding-top: 20px;
+  padding-bottom: 20px;
+  background-color: transparent;
+  border: 0.5px solid rgba(128, 128, 128, 0.363);
+  margin: 10px;
+}
+.badge-button {
+  position: relative;
+  margin: 10px;
+  color: var(--black-color);
+  background-color: transparent;
+  border: none;
+  outline: none;
+  border-radius: 50%;
+}
+.badge-items .kitty {
+  padding: 10px;
+}
+.badge-avatar {
+  position: relative;
+  align-items: flex-end;
+  justify-content: center;
+  margin: 25px;
+}
+.icon-avatar-badge {
+  position: absolute;
+  bottom: 0.5rem;
+  right: 0rem;
+  background-color: var(--green-color);
+  color: var(--white-color);
```

Comment:
there is some repetitive code, maybe you can create a common class for them and include the repetitive styles in that.

Reply:
Thank you for the review!